Bookcrossing.Contracts/Exceptions/ModelExceptions/ErrorDetails.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Newtonsoft.Json;
+
+namespace Bookcrossing.Contracts.Exceptions.ModelExceptions
+{
+    public class ErrorDetails
+    {
+        public int StatusCode { get; set; }
+        public string Message { get; set; }
+        public override string ToString() => JsonConvert.SerializeObject(this);
```

Comment:
Check 'records'.
As far as I remember they have the same ToString implementation under the hood.